TELCODOCS-1931 Describe source IP NAT for MetalLB return traffic (#85119)
openshift/openshift-docs: modules/nw-metallb-when-metallb.adoc

You must configure your networking infrastructure to ensure that network traffic for the external IP address is routed from clients to the host network for the cluster.

After deploying MetalLB with the MetalLB Operator, when you add a service of type `LoadBalancer`, MetalLB provides a platform-native load balancer. 

> When external traffic enters your {product-title} cluster through a MetalLB LoadBalancer service, the return traffic to the client will have the LoadBalancer's external IP address as the source IP.

Reviewer: ```suggestion
When external traffic enters your {product-title} cluster through a MetalLB `LoadBalancer` service, the return traffic to the client has the external IP address of the load balancer as the source IP.
```
See the ISG on [verb tense](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) and the docs guidelines on [general](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) vs specific [API object references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references).

Also, the ISG says [to avoid using possessives](https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives) with inanimate objects.

> When external (ingress) traffic enters your {product-title} cluster through a MetalLB LoadBalancer service, MetalLB configures the Source Network Address Translation (SNAT) for the return (egress) traffic to use the service’s IP address. This ensures that when the traffic exits the cluster back to the client, it appears to come from the service’s external IP address rather than the node IP address.

Reviewer 2: technically, it's not metallb to do this. And it's a bit more complex than that (because it's a reply on a session with a dnat towards the clusterip). I don't think it's important in this scenario. Can we just mention that the return traffic will have the loadbalancer ip as source ip without going into details on how this is done?
